submissions/adeyemiJuwon/php/converter.php

```diff
@@ -0,0 +1,28 @@
+<?php 
+
+
+ function converter($value, $unit) {
+        $pound = 0.4536;
+        $yard = 0.9144;
+        $fahrenheit = ($value * 9/5) + 32;
```

Comment:
move this calculation its block in the `elseif` block

Reply:
Okay Done